Gemfile

```diff
@@ -16,7 +16,7 @@ gem 'puma', '~> 3.7'
 gem 'jbuilder', '~> 2.5'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-gem 'faucet_pipeline_rails'
+gem 'faucet_pipeline_rails', path: "/home/vagrant/Code/faucet_pipeline_rails"
```

Comment:
wat?

Reply:
This was for local testing while it was unreleased. It is gone now.